app/Party.php

```diff
@@ -968,11 +790,6 @@ public function associatedNetworkCoordinators()
         return $coordinators;
     }
 
-    public function getMaxUpdatedAtDevicesUpdatedAtAttribute()
```

Comment:
Definitely not used?

Reply:
Nope - search for MaxUpdatedAtDevicesUpdatedAt finds nowt.